tests/tests_unit/test_api_client.py

```diff
@@ -1189,6 +1189,7 @@ def convert_resource_to_patch_object_test_cases():
             "update": {
                 "name": {"set": "bla"},
                 "unit": {"setNull": True},
+                "unitExternalId": {"setNull": True},
```

Comment:
Guess we could remove this test:
`id="Replace TimeSeries and ignore beta property",`

Reply:
remove the test (lines 1183-1200) or keep? Same for me

Reply:
Sorry, I was unclear, I meant the `pytest.param` with id="Replace TimeSeries and ignore beta property". You decide 😄